src/js/profile/mobile/widget/Expandable.js

```diff
@@ -476,7 +476,11 @@
 				if (element.classList.contains(classes.expandFrom)) {
 					ui.expandFrom = element;
 					ui.expandTo = self._createExpandTo(element);
-					ui.expandButton = self._createExpandButton(ui.expandTo);
+					ui.expandButton = element.querySelector("." + classes.expandButton) ||
```

Comment:
Isn't it nicer if you assign  `"." + classes.expandButton` expression to another variable?

Reply:
Thanks for suggestion. I made the changes.